cli/command/container/hijack.go

```diff
@@ -19,41 +19,40 @@ import (
 // TODO: This could be moved to `pkg/term`.
 var defaultEscapeKeys = []byte{16, 17}
 
-// A hijackedIOStreamer handles copying input to and output from streams to the
+// HijackOptions handles copying input to and output from streams to the
 // connection.
-type hijackedIOStreamer struct {
-	streams      command.Streams
-	inputStream  io.ReadCloser
-	outputStream io.Writer
-	errorStream  io.Writer
-
-	resp types.HijackedResponse
+type HijackOptions struct {
+	resp       types.HijackedResponse
+	ttyInput   io.ReadCloser
+	inputFlag  bool
+	outputFlag bool
+	errorFlag  bool
 
 	tty        bool
 	detachKeys string
 }
```

Comment:
Exporting this type isn't going to be very helpful unless you also export all of the struct fields 😉

Reply:
Ah good catch. Forgot to capitalize the fields as well.